packages/teams-js/src/internal/validOrigins.ts

```diff
@@ -5,6 +5,7 @@ import { inServerSideRenderingEnvironment, isValidHttpsURL } from './utils';
 
 let validOriginsCache: string[] = [];
 const validateOriginLogger = getLogger('validateOrigin');
+const ORIGIN_LIST_TIMEOUT = 1500;
```

Comment:
>ORIGIN_LIST_TIMEOUT [](http://example.com/codeflow?start=6&length=19)

Can you add a unit to this symbol name? E.g., `ORIGIN_LIST_TIMEOUT_IN_MS`

Reply:
Great suggestion, I've added it in